swagger-spec/users/addons_list.yaml

```diff
@@ -1 +1,155 @@
 # /users/{user_id}/addons/
+get:
+  summary: List all addons
+  description: >-
+
+    A paginated list of addons authoried by this user
+
+    #### Returns
+
+    Returns a JSON object containing `data` and `links` keys.
+
+
+    The `data` key contains an array of up to 10 addons.
+    Each resource in the array is a separate addon object.
+
+
+    The `links` key contains a dictionary of links that can be used
+    for [pagination](#Introduction_pagination).
+
+  parameters:
+    - in: path
+      type: string
+      name: user_id
+      required: true
+      description: 'The unique identifier of the user.'
+
+
+  tags:
+    - Users
+  operationId: users_addons_list
+  x-response-schema: Addon
+  responses:
+    '200':
+      description: OK
+      schema:
+        type: array
+        items:
+          $ref: '../addons/definition.yaml'
```

Comment:
This endpoint actually doesn't return true addon objects. I think you'll need to add another model definition that will be a "User Addon" object. 

Compare the `/v2/addons/` response to the `/v2/users/me/addons/` response to see what I mean.